websocket/frontend/src/todolist/Todoform.js

```diff
@@ -36,24 +43,46 @@ class TodoForm extends Component {
     });
   };
 
+  handleBodyChange(event) {
+    this.setState({
```

Comment:
If you do like this, what happens to `task` in `state`?

Reply:
it remains empty until handleTaskChange is executed